readme

NathanSegerlind opened the pull request:
Initial README.md for oni-lda-c version 1.0.1

NathanSegerlind committed:
readme

Initial README.md for oni-lda-c version 1.0.1

NathanSegerlind committed:
readme

added note on loading data to all hosts

rabarona commented on README.md:
````diff
@@ -1,2 +1,195 @@
+Compilation requires that MPI libraries have been installed on the machine where oni-lda-c will be compiled. Other than that, just make in the top oni-lda-c directory:
+```
+[user@system ~]$ cd oni-lda-c
+[user@system  oni-lda-c]$ make
````
Current wiki indicates to run make clean before make, is make clean not required anymore? 
https://github.com/Open-Network-Insight/open-network-insight/wiki/Install%20and%20Configure%20ML

rabarona commented on README.md:
```diff
@@ -1,2 +1,195 @@
+**final.beta** in which each row corresponds to a topic and each row contains the natural logarithm of the probability of each word given that topic.
+
+For example,  if in topic 1, the probability of word 1 were 0.1, the probability of word 2 were 0.89 and the probability of word 3 were 0.01, then
+then the first line of final.beta would be  −2.302585093 −0.116533816 −4.605170186
```
"then" written twice.

rabarona commented:
Very clear and complete.

NathanSegerlind commented:
thanks, man

NathanSegerlind committed:
readme

Incorporated review comments:

- line breaks in maintainers list
- typos
- make clean ; make ... not just make

NathanSegerlind committed:
readme

expanded example to include both netflow and DNS word formation

natedogs911 commented on README.md:
```diff
@@ -1,2 +1,204 @@
+
+Parallel implementation of latent Dirchlet allocation for OpenNetworkInsight, version 1.0.1
+
+oni-lda-c is a program for *topic modelling*:  From a collection documents and integer k, infer k topics and assign to each document a mixture of topics and to each topic a mixture of words. 
```
should this be "From a collection _of_ documents"?

NathanSegerlind replied:
yes, yes it should be "a collection **of** documents"

natedogs911 commented on README.md:
```diff
@@ -1,2 +1,204 @@
+
+oni-lda-c is a program for *topic modelling*:  From a collection documents and integer k, infer k topics and assign to each document a mixture of topics and to each topic a mixture of words. 
+
+Topic modelling is used in the OpenNetworkInsights suspicious connections analysis to generate profiles of typical traffic on the network and estimate
```
"OpenNetworkInsights" should be change to "Open-Network-Insight"

natedogs911 commented on README.md:
```diff
@@ -1,2 +1,204 @@
+
+For example, in a netflow analysis the document is communication between two IP addresses and
+and the "words" are summaries of the netflow records between the two. The
+"topics" are then profiles common traffic on the network, the mixture of topic to the documents
```
profiles of common traffic

natedogs911 commented:
i actually found it very enlightening, perhaps there could be a coffee table paper edition?

NathanSegerlind commented:
we need more full page color photographs for the coffee  table version

NathanSegerlind committed:
readme

incorporated code review comments (typos, formatting of Open-Network-Insight)

NathanSegerlind committed:
readme

Fixed "then then" typo

NathanSegerlind committed:
readme

in the make example:

make -> make clean ; make
